python/lsst/pipe/base/pipelineIR.py

```diff
@@ -373,13 +476,31 @@ def process_args(argument: Union[str, dict]) -> dict:
 
         # integrate any imported pipelines
         accumulate_tasks = {}
+        accumulate_labeled_subsets = {}
         for other_pipeline in self.inherits:
             tmp_IR = other_pipeline.toPipelineIR()
             if accumulate_tasks.keys() & tmp_IR.tasks.keys():
                 raise ValueError("Task labels in the imported pipelines must "
                                  "be unique")
             accumulate_tasks.update(tmp_IR.tasks)
             self.contracts.extend(tmp_IR.contracts)
+            # verify that tmp_IR has unique labels for named subset among
+            # existing labeled subsets, and with existing task labels.
+            overlapping_subsets = accumulate_labeled_subsets.keys() & tmp_IR.labeled_subsets.keys()
+            task_subset_overlap = (accumulate_labeled_subsets.keys() | tmp_IR.labeled_subsets.keys()) \
+                & accumulate_tasks.keys()
```

Comment:
An extra parentheses for line continuation [is preferred](https://developer.lsst.io/python/style.html#python-s-implied-continuation-inside-parens-brackets-and-braces-should-be-used-for-wrapped-lines) over backslash.